### PR TITLE
feat(bsky): init

### PR DESCRIPTION
**Pull request by RoootTheFox**
## 🎉 Theme for [Bluesky](https://bsky.app) 🎉

Bluesky Social is a microblogging social media platform.

## 💬 Additional Comments 💬

Theming this wasn't very hard, although its possible that I missed some stuff.
This is also my first time making a userstyle using LESS (and contributing to userstyles) so there might be inconsistencies(?).

image
image

Latte works, but you have to have Bluesky set to dark mode and set Latte as dark mode theme, because it doesn't seem to
respect `@media (prefers-color-scheme: dark)`?
I haven't found any major visual issues using that as workaround though.

## 🗒 Checklist 🗒

- [x] I have read and followed Catppuccin's [submission guidelines](https://github.com/catppuccin/userstyles/blob/main/docs/userstyle-creation.md).
- [x] I have made a new directory underneath `/styles/<name-of-website>` containing the contents of the [`/template`](https://github.com/catppuccin/userstyles/blob/main/template/) directory.
  - [x] I have ensured that the new directory is in **lower-kebab-case**.
  - [x] I have followed the template and kept the preprocessor as [LESS](https://lesscss.org/#overview).
- [x] I have made sure to update the
      [`userstyles.yml`](https://github.com/catppuccin/userstyles/blob/main/src/userstyles.yml)
      file with information about the new userstyle.
- [x] I have included the following files:
  - `catppuccin.user.css` - all the CSS for the userstyle, based on the
    template.
  - `assets/mocha.webp`, `assets/macchiato.webp`, `assets/frappe.webp` and
    `assets/latte.webp` - individual screenshots of the themed website, in all 4
    Catppuccin flavors.
  - `catwalk.webp` - image of all individual screenshots stitched together,
    generated via [catwalk](https://github.com/catppuccin/toolbox#-catwalk).

### CODE REVIEW

**Review by isabelroses — DISMISSED** [review]
Hello, amazing work so far,

One thing I notice, looking at this theme is that a large amount of the buttons are unthemed. (see below)
image
image

**Comment by RoootTheFox** [comment]
What color should the buttons use? Im thinking either `text` or `subtext1` since these are the closest to the original color.

**Comment by RoootTheFox** [comment]
subtext1:
subtext1
text:
text

**Comment by RoootTheFox** [comment]
I think I themed pretty much everything now (I don't know how many times I thought this already but whatever lmao)
I also fixed some contrast issues and other things that broke/looked off

**Comment by RoootTheFox** [comment]
opinions needed on this notification style for mentions/highlights (background color) (currently `fadeout(@accent-color, 70%)`)
image

File: styles/bsky/catppuccin.user.css [hunk 1]
```diff
@@ -0,0 +1,331 @@
...
+    #catppuccin(@darkFlavor, @accentColor);
+  }
+
+  /* manually set colorscheme */
+  .light-theme {
+    #catppuccin(@lightFlavor, @accentColor);
+  }
+  .dark-theme {
+    #catppuccin(@darkFlavor, @accentColor);
+  }
```
Comment (uncenter) [inline styles/bsky/catppuccin.user.css:330]:
```suggestion
```

Unless these classes are actually used for dark/light mode you can delete this.

Comment (RoootTheFox) [reply styles/bsky/catppuccin.user.css:330]: supporting bsky's light mode would essentially mean rewriting half(?) of the userstyle, would it be fine if i just instruct people to set their bsky to dark mode even if they want to use latte? (and make the userstyle only have one theme toggle)

Comment (RoootTheFox) [reply styles/bsky/catppuccin.user.css:330]: (for context, latte seems to work perfectly fine with bsky set to dark mode)

Comment (RoootTheFox) [reply styles/bsky/catppuccin.user.css:330]: also @uncenter im able to provide you with an invite code if you need one (for testing the theme or general usage)

**Comment by isabelroses** [comment]
@RoootTheFox can I convert this to a draft as it seems to be a bit WIP

File: styles/bsky/catppuccin.user.css (hunk 1 above)
Comment (uncenter) [reply styles/bsky/catppuccin.user.css:340]: That would be helpful, thank you.

**Comment by uncenter** [comment]
> opinions needed on this notification style for mentions/highlights (background color) (currently `fadeout(@accent-color, 70%)`) image

Looks a little too prominent/visible to me... what does it look like normally (unthemed)?

**Comment by RoootTheFox** [comment]
> > opinions needed on this notification style for mentions/highlights (background color) (currently `fadeout(@accent-color, 70%)`) image
> 
> Looks a little too prominent/visible to me... what does it look like normally (unthemed)?

normally its a very dark blue
image

**Comment by RoootTheFox** [comment]
> @RoootTheFox can I convert this to a draft as it seems to be a bit WIP

sure

**Comment by uncenter** [comment]
> > > opinions needed on this notification style for mentions/highlights (background color) (currently `fadeout(@accent-color, 70%)`) image
> > 
> > 
> > Looks a little too prominent/visible to me... what does it look like normally (unthemed)?
> 
> normally its a very dark blue image

Hmm the contrast feels off.

**Comment by isabelroses** [comment]
@RoootTheFox just letting you know I now believe the theme to be in a state that is possible to merge, I just want to check with you whether your still willing to maintain the theme.

Thanks for your hard work and time.

**Comment by RoootTheFox** [comment]
@isabelroses i am still willing to maintain the theme, although i'm not sure how much time i will be able to dedicate to it, i will try my best though

**Comment by isabelroses** [comment]
> although i'm not sure how much time i will be able to dedicate to it, i will try my best though

That's alright, if you need help, or ever want to stop just let us know.

**Review by isabelroses — APPROVED** [review]

**Pull request merged**